modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/login-button/index.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable no-undef */
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+const loginButton = fragmentElement.querySelector('#evp-login-button');
```

Comment:
I also don't think `evp-login-button` a good name, since is not applied to a <button> element, this is confuse

```suggestion
const loginButton = fragmentElement.querySelector('.evp-login-button a');
```